release-notes/opensearch-release-notes-2.7.0.md

```diff
@@ -0,0 +1,642 @@
+* Searchable snapshots also move from experimental to production-ready with this release, allowing users to search indexes that are stored as snapshots within remote repositories in real time.
+* The third feature moving out of experimental status is support for multiple data sources in OpenSearch Dashboards. Dynamically manage data sources across multiple OpenSearch clusters, combine visualizations into a single dashboard, and more.
+* New flat object fields let you store complex JSON objects in an index without indexing all subfields separately, offering improved resource utilization.
+* You can now access observability features from OpenSearch Dashboards, create your own observability dashboards, add event analytics visualizations, and more without leaving the Dashboards environment.
+* Geospatial functionality receives upgrades as new shape-based filters let you filter your geospatial data by drawing a rectangle or polygon over a selected area of the map. This release also enables OpenSearch to display maps in local languages.
+* OpenSearch makes managing multiple indexes simpler with the addition of component templates. These templates can help you abstract common index settings, mappings, and aliases into reusable building blocks.
+* Another time-saving upgrade for OpenSearch administrators comes with the availability of dynamic tenants management in OpenSearch Dashboards. Admins can view, configure, and enable or disable tenancy within Dashboards and effect those changes without a restart.
+* You can now use the Performance Analyzer plugin to identify hot shards within an index, so you can take action and mitigate potential impacts on performance and availability.
+
+### Experimental Features
+OpenSearch 2.7.0 includes the following experimental features. Experimental features are disabled by default. For instructions on how to enable them, refer to the documentation for the feature.
+
+* A new correlation engine is added to the Security Analytics toolkit, enabling a knowledge graph that can be used to identify, store, and recall connected events data to help you identify patterns and investigate relationships.
+* The experimental ML Framework gains a new automatic reloading mechanism for ML models, allowing you to auto-reload deployed models when a cluster restarts or when a node rejoins the cluster.
+
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 2.7.0](https://opensearch.org/versions/opensearch-2-7-0.html) includes the following feature, enhancement, bug fixes, infrastructure, documentation, maintenance and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-2.7.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-2.7.0.md).
+
```

Comment:
Can we add a link to the documentation release notes? https://github.com/opensearch-project/documentation-website/blob/main/release-notes/opensearch-documentation-release-notes-2.7.0.md

Reply:
I'm good with this. @zelinh ?

Reply:
@hdhalter @jhmcintyre Thanks for reviewing. I have added documentation release notes link below; and also updated based on reviews.